apollo-compiler/src/test/graphql/com/example/HeroNameDirectiveExpected.java

```diff
@@ -0,0 +1,12 @@
+package com.example;
```

Comment:
the paths for the test files has been changed on #27, so this will cause a conflict. Just a heads up